### ISSUE TITLE
recursion errors with multiple checkpoints

### ISSUE THREAD

**Issue by NeonLightning** [audioscavenger/save-image-extended-comfyui#16]
i'm trying to do outpainting and use a second checkpoint for the outpainting and one for the base image
i'm using this as my folder name 25.ckpt_name, 50.ckpt_name, 31.lora_name, 30.lora_name,
it's giving me an error about recursion

```
SaveImageExtended 2.76 error: An error occurred while creating the subfolder or saving the image: [WinError 267] The directory name is invalid: "D:\\ai\\stabilitymatrix\\Data\\Packages\\ComfyUI\\output\\xl\\enjoyXLSuperRealistic_v50lightning-{'content': 'xl"
!!! Exception during processing!!! object of type 'NoneType' has no len()
Traceback (most recent call last):
  File "D:\ai\stabilitymatrix\Data\Packages\ComfyUI\execution.py", line 151, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
  File "D:\ai\stabilitymatrix\Data\Packages\ComfyUI\execution.py", line 95, in get_output_data
    output_is_list = [False] * len(results[0])
TypeError: object of type 'NoneType' has no len()
```

**Comment by audioscavenger** [comment]
You are the second person to report this. is the checkpoint loader a new node from Pythongossss?
Same fix as the other open issue, will detect and parse json when it happens. Will have a look in a few day, sorry. 

I just finished remodeling and moving in, exhausted. give me a few. please.

**Comment by audioscavenger** [comment]
hey please post your picture or workflow picture here, thanks

**Comment by audioscavenger** [comment]
this comes from pythongosssss custom scripts, fixed in v2.81
